source/help/apps/desktop-guide.rst

```diff
@@ -40,108 +40,95 @@ Editing servers and teams
++===============================================================================+===========================+===========================+===========================+
+| Allow mixed content                                                           | Yes                       | Yes                       | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Show red badge on taskbar icon to indicate unread messages                    | Yes                       | Yes                       |                           |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Flash taskbar icon when a new message is received                             | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Start app on login                                                            | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Hide menu bar                                                                 | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Toggle window visibility when clicking on the tray icon                       | Yes                       |                           |                           |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Show icon on menu bar                                                         |                           | Yes                       | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Leave app running in notification center when application window is closed    |                           | Yes                       | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
 
```

Comment:
If you want to quickly skim through Mac-specific options, how long would it take to find them on the definition list?

If it's easy, then yeah I guess the table is unnecessary.

The original intent of the table was to have a quick reference at the top, followed by more information after the table if needed.